hasher-matcher-actioner/hmalib/models.py

```diff
@@ -48,6 +56,90 @@ def from_sns_message(cls, message: str) -> "SNSMessage":
         raise NotImplementedError
 
 
+@dataclass
+class SignalMetadataBase(DynamoDBItem):
+    """
+    Base for signal metadata.
```

Comment:
blocking: Would be super-hard to change prefixes once a database is built. Can we use source agnostic nomenclature NOW?

Reply:
sure are you okay with 'ds#' for dataset or have a specific one in mind?